pycbc/distributions/mass.py

```diff
@@ -0,0 +1,194 @@
+# Copyright (C) 2021 Yifan Wang
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+""" This modules provides classes for evaluating distributions for mchirp and 
+mass ratio from uniform component mass.
+"""
+
+import numpy
+from pycbc.distributions import power_law
+from pycbc.distributions import bounded
+from scipy.interpolate import interp1d
+from scipy.special import hyp2f1
+
+class MchirpfromUniformMass1Mass2(power_law.UniformPowerLaw):
+    """ For a uniform distribution in volume using spherical coordinates, this
+    is the distriubtion to use for the radius.
+
+    For more details see UniformPowerLaw.
+    """
+    name = "mchirp_from_uniform_mass1_mass2"
+    def __init__(self, dim=2, **params):
+        super(MchirpfromUniformMass1Mass2, self).__init__(dim=2, **params)
+
+class QfromUniformMass1Mass2(bounded.BoundedDist):
+    """
+    A distribution for mass ratio (i.e., q) from uniform component mass. The parameters are
```

Comment:
Put the first sentence on the same line as the `"""`. Then put a blank line to separate that from the rest.